CHANGELOG.md

```diff
@@ -20,12 +20,13 @@
 - Added make integration command
 - Upgrade RabbitMQ Docker image to 3.13
 - Updated README with steps on running the processes locally
-
+- Made new config local.yaml file to run in docker compose file
```

Comment:
The local.yaml file should be used to setup the environment variables when running processes locally.

Reply:
That makes sense and I'll change the description here to reflect that.